src/CoreWCF.NetTcp/src/CoreWCF/Channels/Framing/NetMessageFramingConnectionHandler.cs

```diff
@@ -97,7 +98,22 @@ internal static UriPrefixTable<HandshakeDelegate> BuildAddressTable(IServiceProv
                         continue;
                     }
 
-                    HandshakeDelegate handshake = BuildHandshakeDelegateForDispatcher(dispatcher);
+                    IServiceDispatcher _serviceDispatcher = null;
+                    if (!(dispatcher.Binding is CustomBinding binding))
+                    {
```

Comment:
variable declaration of `binding` here is unused as the if block is skipped if it is a custom binding so the variable is only in scope when it's null.

Reply:
removed